src/integer_mod_q/z_q/from.rs

```diff
@@ -108,6 +108,42 @@ impl From<&Zq> for Zq {
     }
 }
 
+impl Zq {
+    /// Create a [`Zq`] integer from a [`String`], i.e. its UTF8-Encoding.
+    /// The inverse of this function is [`Zq::to_utf8`].
+    ///
+    /// Parameters:
+    /// - `message`: specifies the message that is transformed via its UTF8-Encoding
+    ///   to a new [`Zq`] instance.
+    /// - `modulus`: Defines the modulus by which `value` is reduced.
+    ///
+    /// Returns value defined by `message` mod `modulus` as [`Zq`].
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::integer_mod_q::Zq;
+    /// let message = "hello!";
+    ///  
+    /// let value = Zq::from_utf8(&message, i64::MAX).unwrap();
+    /// assert_eq!(Zq::from((36762444129640u64, i64::MAX)), value);
+    /// ```
+    ///
+    /// # Panics ...
+    /// - if `modulus` is smaller than `2`.
```

Comment:
```suggestion
    /// ```
    ///
    /// # Errors and Failures
    /// - Returns a [`ConversionError`](MathError::ConversionError) if the provided modulus 
    ///     is smaller than the UTF8-Encoding of the message.
    ///
    /// # Panics ...
    /// - if `modulus` is smaller than `2`.
```